packages/relay/src/lib/errors/JsonRpcError.ts

```diff
@@ -237,4 +237,9 @@ export const predefined = {
     code: -39012,
     message: 'Unknown block'
   }),
+  'INVALID_BLOCK_RANGE': new JsonRpcError({
```

Comment:
If the range is too large or invalid. Might be good to add a custom message to distinguish when say toBlock is before fromBlock vs when the range is too large

Reply:
This case is already covered and there is a test for it: https://github.com/hashgraph/hedera-json-rpc-relay/pull/1595#discussion_r1284222480